test/test_framework.py

```diff
@@ -894,6 +894,9 @@ def test_stored_dict_repr(self):
         self.assertEqual(repr(StoredDict(None, {})), "ops.framework.StoredDict()")
         self.assertEqual(repr(StoredDict(None, {"a": 1})), "ops.framework.StoredDict({'a': 1})")
 
+    def test_stored_dict_copy(self):
+        self.assertEqual(StoredDict(None, {"a": 1}).copy(), {"a": 1})
```

Comment:
We should probably also have a test that modifying a key in the original StoredDict doesn't modify the copy.

Reply:
Done (and also verifies that modifying the copy doesn't modify the original StoredDict)